CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Bugfixes
 
-* None.
+* Fix an issue where spurious crashes might occur when the host application is
```

Comment:
It's not accurate to say the crashes are spurious. They're real crashes.

I'd phase this more like:
> Fix a crash that could occur if new Realm instances were created while the application was exiting.